clarifai/cli/model.py

```diff
@@ -68,3 +69,130 @@ def run_locally(model_path):
+    nodepool_id = config.get('nodepool_id', nodepool_id)
+    deployment_id = config.get('deployment_id', deployment_id)
+    inference_params = config.get('inference_params', inference_params)
+    output_config = config.get('output_config', output_config)
+  if sum([1 for opt in [model_id, model_url] if opt]) != 1:
+    raise ValueError("Exactly one of --model_id or --model_url must be provided.")
+  if sum([1 for opt in [file_path, url, bytes, input_id] if opt]) != 1:
+    raise ValueError("Exactly one of --file_path, --url, --bytes or --input_id must be provided.")
+  if sum([
+      opt[1] for opt in [(compute_cluster_id, 0.5), (nodepool_id, 0.5), (deployment_id, 1)]
+      if opt[0]
+  ]) != 1:
+    raise ValueError(
+        "Either --compute_cluster_id & --nodepool_id or --deployment_id must be provided.")
+  if model_url:
+    model = Model(url=model_url, pat=ctx.obj['pat'], base_url=ctx.obj['base_url'])
+  else:
+    model = Model(
+        model_id=model_id,
+        user_id=user_id,
+        app_id=app_id,
+        pat=ctx.obj['pat'],
+        base_url=ctx.obj['base_url'])
+
+  if inference_params:
+    inference_params = json.loads(inference_params)
+  if output_config:
+    output_config = json.loads(output_config)
+
+  if file_path:
+    model_prediction = model.predict_by_filepath(
+        filepath=file_path,
+        input_type=input_type,
+        compute_cluster_id=compute_cluster_id,
+        nodepool_id=nodepool_id,
+        deployment_id=deployment_id,
+        inference_params=inference_params,
+        output_config=output_config)
+  elif url:
+    model_prediction = model.predict_by_url(
+        url=url,
+        input_type=input_type,
+        compute_cluster_id=compute_cluster_id,
+        nodepool_id=nodepool_id,
+        deployment_id=deployment_id,
+        inference_params=inference_params,
+        output_config=output_config)
+  elif bytes:
+    bytes = str.encode(bytes)
+    model_prediction = model.predict_by_bytes(
+        input_bytes=bytes,
+        input_type=input_type,
+        compute_cluster_id=compute_cluster_id,
+        nodepool_id=nodepool_id,
+        deployment_id=deployment_id,
+        inference_params=inference_params,
+        output_config=output_config)
+  elif input_id:
+    inputs = [Input.get_input(input_id)]
+    runner_selector = None
+    if deployment_id:
+      runner_selector = Deployment.get_runner_selector(
+          user_id=ctx.obj['user_id'], deployment_id=deployment_id)
+    elif compute_cluster_id and nodepool_id:
+      runner_selector = Nodepool.get_runner_selector(
+          user_id=ctx.obj['user_id'],
+          compute_cluster_id=compute_cluster_id,
+          nodepool_id=nodepool_id)
+    model_prediction = model.predict(
+        inputs=inputs,
+        runner_selector=runner_selector,
+        inference_params=inference_params,
+        output_config=output_config)
+  click.echo(f"Prediction successful:\n{model_prediction}")
```

Comment:
There is might be the case where prediction failed, I think we can just print `model_prediction`
`click.echo(model_prediction)`

Reply:
This is fine I think, If model prediction fails, [this](https://github.com/Clarifai/clarifai-python/blob/282238dc8aba7b25811c819c619d510c760f2d25/clarifai/client/model.py#L448) will take care of the failure message I believe.